6_gif_tasks.yml

```diff
@@ -3252,7 +3252,7 @@ targets:
   # --- a_20170901_23 --- #
   
   storm_point_a_20170901_23:
-    command: prep_storm_point_fun('2_process/out/storm_points_interp.rds.ind', I('2017-09-01 23:00:00'))
+    command: prep_storm_point_fun('2_process/out/storm_points_interp.rds.ind', I('2017-09-01 23:00:00'), hurricane_col)
```

Comment:
we should maybe think about removing this task .yml from the repo...or not...it's nice to be able to see, but also pretty unweildy....